nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/provenance/nf-provenance.js

```diff
@@ -223,11 +227,32 @@ nf.Provenance = (function () {
                         });
                     }
 
-                    // once the table is initialized, finish initializing the page
-                    initializeProvenancePage().done(function () {
+                    var setBodySize = function () {
+                        $('body').css({
+                            'height': $(window).height() + 'px',
+                            'width': $(window).width() + 'px'
+                        });
+
+                        var provenanceTableHeight = $('#provenance').outerHeight() -
+                            $('#provenance-header-text').outerHeight() -
+                            $('#provenance-refresh-container').outerHeight() - 20; //20 for padding at bottom of table
+                        if (top === window) {
+                            provenanceTableHeight -= 80; //80 for padding
+                        }
+                        $('#provenance-table').height(provenanceTableHeight).show();
```

Comment:
I think this logic is unnecessary as the provenance table is bound from the bottom using position absolute. The height should adjust due to the CSS rules.